pynamodb/attributes.py

```diff
@@ -239,6 +239,48 @@ def deserialize(self, value):
         return json.loads(value)
 
 
+class NumberListAttribute(Attribute):
+    """
+    This is a list attribute that supports only numbers (i.e. integers) or
+    lists of numbers.
+    
+    The DynamoDB List attribute does actually support mixed attribute types,
+    but this one only supports numbers and lists of numbers. Using non-integers
+    or lists of non-integers with this type will produce undefined results.
+    """
+    attr_type = LIST
+
+    @staticmethod
+    def serialize(values):
+        """
+        Encode the given list of numbers into a list of AttributeValue types.
+        """
+        rval = []
+        for v in values:
+            if type(v) is list:
+                rval += [{'L': NumberListAttribute.serialize(v)}]
```

Comment:
Would you mind using the constants for 'L' and 'N' here to keep the style consistent?